packtools/sps/formats/sps_xml/article_meta.py

```diff
@@ -0,0 +1,224 @@
+    """
+    if article_title_text := data.get("article-title"):
+        title_group_elem = ET.SubElement(article_meta, "title-group")
+        ET.SubElement(title_group_elem, "article-title").text = article_title_text
+
+        trans_title_dict = data.get("trans-title", {})
+        for lang, text in trans_title_dict.items():
+            trans_title_group_elem = ET.SubElement(
+                title_group_elem, "trans-title-group", attrib={"xml:lang": lang}
+            )
+            ET.SubElement(trans_title_group_elem, "trans-title").text = text
+
+
+def process_contribs(node, article_meta):
+    """
+    Adiciona o grupo de contribuições ao `article-meta`.
+
+    Args:
+        node (dict): Dicionário contendo o grupo de contribuições.
+            Exemplo:
+            {
+                "contrib-group": [<Element>, <Element>]
+            }
+        article_meta (xml.etree.ElementTree.Element): Elemento 'article-meta' onde
+                                                      o grupo de contribuições será adicionado.
+    """
+    if contribs := node.get("contrib-group"):
+        contrib_group_elem = ET.SubElement(article_meta, "contrib-group")
+        for contrib in contribs:
+            contrib_group_elem.append(contrib)
+
+
+def process_standard_items(data, article_meta):
+    """
+    Adiciona elementos padrão como volume, issue, fpage e lpage ao 'article-meta'.
+
+    Args:
+        data (dict): Dicionário contendo os valores dos itens padrão.
+            Exemplo:
+            {
+                "volume": "69",
+                "issue": "3",
+                "fpage": "227",
+                "lpage": "232"
+            }
+        article_meta (xml.etree.ElementTree.Element): Elemento 'article-meta' onde
+                                                      os itens padrão serão adicionados.
+    """
+    for item in ("volume", "issue", "fpage", "lpage"):
+        if elem_text := data.get(item):
+            ET.SubElement(article_meta, item).text = elem_text
+
+
+def process_node_elements(node, article_meta, keys):
+    """
+    Adiciona elementos específicos ao 'article-meta' com base em uma lista de chaves.
+
+    Args:
+        node (dict): Dicionário contendo os elementos a serem adicionados.
+            Exemplo:
+            {
+                "affs": [<Element>, <Element>],
+                "author-notes": <Element>,
+                "pub-dates": [<Element>, <Element>]
+            }
+        article_meta (xml.etree.ElementTree.Element): Elemento 'article-meta' onde
+                                                      os elementos serão adicionados.
+        keys (tuple): Lista de chaves para elementos a serem adicionados ao 'article-meta'.
+    """
+    for elem_name in keys:
+        if elements := node.get(elem_name):
+            if isinstance(elements, list):
+                for elem in elements:
+                    article_meta.append(elem)
```

Comment:
@Rossi-Luciano você tentou usar o article_meta.extent(elem) no lugar de usar for + append?